server/api.js

```diff
@@ -13,8 +13,18 @@ router.get("/videos", async (_, res) => {
 	}
 });
 
-router.get("/health", async (_, res) => {
-	res.sendStatus(200);
+router.post("/videos", async (req, res) => {
+	if (!req.body.title) {
+		return res.status(422).json({ message: "Title field is required" });
+	}
+	if (!req.body.src) {
+		return res.status(422).json({ message: "src field is required" });
+	}
+	const result = await db.query(
+		`INSERT INTO videos (title,src) VALUES ('${req.body.title}','${req.body.src}') RETURNING id`
```

Comment:
This line works, but sadly it's  horribly insecure thanks to SQL injection. Anyone could delete your videos table just by sending a src with a value like this:
Fakesrc');DROP TABLE videos;

I don't know if Kieth mentioned this, and even if he did, it's easy not to make the connection so there’s no shame in having made the mistake. However, you need to get into the habit of doing  the right thing have a look at  https://stackoverflow.com/questions/58174695/prevent-sql-injection-with-nodejs-and-postgres